neural_style.py

```diff
@@ -102,6 +102,8 @@ def build_parser():
     parser.add_argument('--pooling',
             dest='pooling', help='pooling layer configuration: max or avg (default %(default)s)',
             metavar='POOLING', default=POOLING)
+    parser.add_argument('--overwrite-image', action='store_true',
```

Comment:
Can we change the argument to just `--overwrite`, rather than `--overwrite-image`?